bindings/wasm/src/account/update/attach_method_relationship.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use js_sys::Promise;
+use serde_repr::Deserialize_repr;
+use serde_repr::Serialize_repr;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::future_to_promise;
+
+use identity::account::Update;
+use identity::account::UpdateError::MissingRequiredField;
+use identity::core::OneOrMany::Many;
+use identity::core::OneOrMany::One;
+use identity::did::MethodRelationship;
+
+use crate::account::wasm_account::WasmAccount;
+use crate::error::wasm_error;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+#[wasm_bindgen(js_class = Account)]
+impl WasmAccount {
+  /// Attaches a single or multiple relationships to the given method, if the method exists.
+  ///
+  /// Note: The method needs to be in the set of verification methods,
+  /// so it cannot be an embedded one.
+  #[wasm_bindgen(js_name = attachMethodRelationships)]
+  pub fn attach_relationships(&mut self, input: &AttachMethodRelationshipOptions) -> Result<Promise> {
```

Comment:
There's a difference here since in TS, it's possible to set an array of relationships, while this is done with a separate method call with the builder pattern in Rust. So making it plural in TS makes more sense in my opinion. Open to change it if people think otherwise.